paket.dependencies

```diff
@@ -1,11 +1,11 @@
-framework: netstandard2.0
+framework: netstandard2.0, net462, netcoreapp2.0
```

Comment:
include these frameworks because we have target outputs (tests and such) that have these frameworks.